invenio_app_ils/ui/src/invenio_app_ils/pages/backoffice/DocumentsSearch/components/ResultsList/ResultsList.js

```diff
@@ -28,7 +28,7 @@ export class ResultsList extends Component {
     const rows = this.prepareData();
     const headerActionComponent = (
       <NewButton
-        text={'New item'}
+        text={'New document'}
```

Comment:
Not sure you need the curly brackets here.

Reply:
if we are going to change that, it should be changed everywhere, we should align and change the syntax settings for everyone.  In react documentation we see this https://reactjs.org/docs/jsx-in-depth.html#string-literals

Reply:
I think we can survive with brackets :)